README.md

```diff
@@ -183,7 +184,9 @@ android.testOptions {
 }
 
 dependencies {
-    testImplementation 'io.kotest:kotest-runner-junit5:<version>'
+    testImplementation 'io.kotest:kotest-runner-junit5:<version>' // for kotest framework
+    testImplementation 'io.kotest:kotest-assertions-core:<version>' // for kotest core assertions
+    testImplementation 'io.kotest:kotest-assertions-jvm:<version>' // for kotest jvm assertions
```

Comment:
We can simplify this to

testImplementation("io.kotest:kotest-runner-junit5-jvm:") // for kotest framework
testImplementation("io.kotest:kotest-assertions-core-jvm:") // for kotest core jvm assertions